tqdm/_tqdm.py

```diff
@@ -83,7 +83,7 @@ def print_status(self, s):
 
 
 def tqdm(iterable, desc=None, total=None, leave=False, file=sys.stderr,
-         ncols=None, mininterval=0.1, miniters=None):
+         ncols=None, mininterval=0.1, miniters=1, disable=False):
```

Comment:
I had changed it in README, since I just copy-pasted it... What did you do with the rebase? :P